python/ml4ir/base/model/relevance_model.py

```diff
@@ -91,6 +95,11 @@ def __init__(
         self.tfrecord_type = tfrecord_type
         self.file_io = file_io
 
+        if len(eval_config) == 0:
+            self.eval_config = self.file_io.read_yaml(RelevanceModelConstants.DEFAULT_EVAL_CONFIG_YAML)
```

Comment:
@mohazahran This should be handled upstream in the pipeline class. The function definition expects a dictionary.

Reply:
I don't follow which function you mean.

Reply:
The if-else. If there is no eval_config, read the default one in the pipeline class and pass it here. RelevanceModel shouldn't be doing this.